packages/frontend/src/pages/Signup.tsx

```diff
@@ -213,6 +213,13 @@ const Signup: FC = () => {
                             >
                                 Privacy Policy.
                             </FooterCta>
+                            and our{' '}
+                            <FooterCta
```

Comment:
We also need this in `packages/frontend/src/pages/Register.tsx`

Reply:
Added!